fixups/RegLegacyFixups/InitializeFixup.cpp

```diff
@@ -256,6 +255,63 @@ void InitializeConfiguration()
 
                             specItem.remediationRecords.push_back(recordItem);
                         }
+
+                        // Look for DeletionMarker remediation
+                        else if (type.compare(L"DeletionMarker") == 0)
+                        {
+                            Log("RegLegacyFixups:      is DeletionMarker\n");
+                            recordItem.remeditaionType = Reg_Remediation_type_DeletionMarker;
+
+                            // Look for hive - data
+                            auto hiveType = regItemObject.try_get("hive")->as_string().wstring();
```

Comment:
try_get returns nullptr if the item can't be found.  This will crash.

Reply:
There is a try catch block from where we are trying to attach the dll to the process(In DllMain of PsfRuntime). These are all mandatory fields, and we don't want to load the fixup if anything fails here.

Reply:
Correct, these are all mandatory field. This will be also covered in public documentation.